### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by dsandoval11**
## Summary
- Navbar component is added to the book list

## Screenshot
chrome-capture (2)

## Trello Card
https://trello.com/c/ybBNVCvx/31-navbar

### CODE REVIEW

File: scss/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@import 'colors';
```
Comment (mcallegari10) [inline scss/navbar.scss:1]: import this in `application.scss` so you don't have to do it every time

File: scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  .logo {
+    height: 100%;
+    max-height: 64px;
+  }
+
+  .logout {
+    font-size: 16px;
+  }
+}
```
Comment (mcallegari10) [inline scss/navbar.scss:27]: Add EOF line

**Review by mcallegari10 — APPROVED** [review]

File: scss/navbar.scss (hunk 1 above)
Comment (dsandoval11) [reply scss/navbar.scss:1]: Fixed in PR #11 😬

File: scss/navbar.scss (hunk 2 above)
Comment (dsandoval11) [reply scss/navbar.scss:27]: Fixed in PR #11 😬

**Pull request merged**